cleanup.sh

```diff
@@ -4,6 +4,7 @@ set -e
 
 apt clean
 rm -rf /var/lib/snapd/cache/*
+rm -rf /tmp/cache
```

Comment:
There are some lines removing specific directories from /tmp, then all of tmp, then some other parts... This is pretty useless, as `/tmp` is stored in RAM (for a very long time now on major distros).

So this line of code is useless twice: it gets deleted 10 lines below, and it is removed already on reboot. (Although, I think that the thing 10 lines below would be happy to break some non-trivial GUI apps)

```bash

# Clear out temporary files
rm -rf /tmp/*
rm -rf /var/tmp/*
```

Maybe revamping this kinds of code could be useful.

Reply:
I would remove this line.

Reply:
I might recall wrong but last time I checked it seemd like /tmp wasn't tmpfs.

Reply:
Actually, if /tmp were a tmpfs I think we shouldn't use it for storing such large files.

Reply:
Yeah, it's not tmpfs, sorry. I have many problems right now accepting this :D How did I not notice in the previous N years?

However, the main point still stands: `/tmp/cache` is already deleted somewhere else.

Reply:
Fair enough.

Reply:
No, because it's generally run at bootup, not at shutdown (and problem is that we are creating an image without starting up the system)